src/site/stages/build/process-cms-exports/schemas/output/node-health_care_region_page.js

```diff
@@ -187,14 +160,9 @@ module.exports = {
     'fieldGovdeliveryIdEmerg',
     'fieldGovdeliveryIdNews',
     'fieldOperatingStatus',
-    'fieldFacebook',
-    'fieldFlickr',
-    'fieldInstagram',
-    'fieldTwitter',
     'fieldNicknameForThisFacility',
     'fieldLinkFacilityEmergList',
     'fieldPressReleaseBlurb',
-    'fieldLeadership',
```

Comment:
Why is this being removed?

Reply:
Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh